docs/06-authentication.md

```diff
@@ -0,0 +1,93 @@
+# Configuring Authentication
+OpenShift 4 installs with only a single cluster superuser, `kubeadmin`. If
```

Comment:
we still have `system:admin`

Reply:
`kubeadmin` is an alias for `system:admin` effectively, isn't it?

If you login as `kubeadmin` and then `oc whoami` doesn't it show `system:admin`?

Reply:
No
```
# oc whoami
kube:admin
```

Reply:
If you do not think this descriptions have no problem,I am fine.